javascript/oauth-sdk-examples/passport/.env

```diff
@@ -0,0 +1,2 @@
+CLIENT_ID=
```

Comment:
Maybe add like placeholder tags like `<YOUR_CLIENT_ID>` and `<YOUR_CLIENT_SECRET>` to make it more clear. But you don't have to.